packages/block-library/src/button/edit.native.js

```diff
@@ -227,12 +219,19 @@ class ButtonEdit extends Component {
 		this.setState( { isLinkSheetVisible: false } );
 	}
 
+	onContainerLayout( { nativeEvent } ) {
```

Comment:
just asking - why you changed the name? :) I guess at some point in you had more than one `onLayout` but i see that there is only one finally :)